sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -21,13 +22,9 @@
 from ._result import Result
 
 
-Arg = Union["Timestream", int, str, float, bool, None]
-"""
-The type of arguments to most Timestream functions.
-May be a Timestream, a literal (int, str, float, bool), or null (`None`).
-"""
```

Comment:
Hm, I kind of liked this conciseness. Maybe could have renamed `Arg` to something a bit more unique, but any reason you want the explicit `Union`?

Reply:
1. The docstring never shows up in Sphinx etc., so I just removed it.
2. I kind of like being able to say "the argument is a timestream or a literal" which is why I changed from `Arg` to `Literal` as the common definition. That way we don't need to think about "what is `Arg`"